sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -649,7 +650,35 @@ def monitoring_infos(self):
       for mi in op.monitoring_infos(transform_id).values():
         fixed_mi = self._fix_output_tags_monitoring_info(transform_id, mi)
         all_monitoring_infos_dict[monitoring_infos.to_key(fixed_mi)] = fixed_mi
-    return list(all_monitoring_infos_dict.values())
+
+    infosList = list(all_monitoring_infos_dict.values())
+
+    def inject_pcollection_into_element_count(monitoring_info):
```

Comment:
Please add a pydoc string, describing the approach used to get the pcollection name.